.github/workflows/build_all.yml

```diff
@@ -9,6 +9,11 @@ on:
   schedule:
     # Run every Sunday at midnight
     - cron: '0 0 * * 0'
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
```

Comment:
build_all runs either on a schedule or manually, unlike sanity_checks which is run on PRs and can be iterated.

I *think* (but am not sure) that setting concurrency would prevent manually triggering this job twice in quick succession, with different values for `fatal_warnings`. It might be better to just drop concurrency from this one job.